esmDependencyTable.csv

```diff
@@ -0,0 +1,158 @@
+morgan,commonjs,1.10.0,Tue Feb 21 2023 02:19:48 GMT+0000 (Greenwich Mean Time),1.10.0,Fri Mar 20 2020 18:00:13 GMT+0000 (Greenwich Mean Time),1285
+jalaali-js,commonjs,1.2.6,Sun Jun 19 2022 04:39:17 GMT+0100 (British Summer Time),1.2.6,Wed May 18 2022 07:40:33 GMT+0100 (British Summer Time),496
+mkdirp,commonjs,3.0.1,Tue May 16 2023 19:48:04 GMT+0100 (British Summer Time),1.0.4,Fri Apr 03 2020 18:03:08 GMT+0100 (British Summer Time),1271
+isomorphic-fetch,commonjs,3.0.0,Sun Jun 19 2022 04:03:37 GMT+0100 (British Summer Time),3.0.0,Wed Sep 23 2020 08:27:09 GMT+0100 (British Summer Time),1098
+moment-timezone,commonjs,0.5.43,Thu May 11 2023 15:15:15 GMT+0100 (British Summer Time),0.5.43,Fri Mar 31 2023 11:12:53 GMT+0100 (British Summer Time),179
+helmet,module,7.0.0,Sat May 06 2023 23:37:55 GMT+0100 (British Summer Time),4.6.0,Sun May 02 2021 16:13:09 GMT+0100 (British Summer Time),877
+react,commonjs,18.3.0-next-fecc288b7-20221025,Mon Sep 25 2023 17:17:27 GMT+0100 (British Summer Time),18.2.0,Tue Jun 14 2022 20:46:38 GMT+0100 (British Summer Time),469
+url-parse,commonjs,1.5.10,Tue Jun 28 2022 07:17:22 GMT+0100 (British Summer Time),1.5.10,Tue Feb 22 2022 15:49:03 GMT+0000 (Greenwich Mean Time),581
+@babel/core,commonjs,8.0.0-alpha.2,Mon Sep 25 2023 09:11:54 GMT+0100 (British Summer Time),7.22.17,Fri Sep 08 2023 14:53:33 GMT+0100 (British Summer Time),18
+@babel/eslint-parser,commonjs,8.0.0-alpha.2,Mon Sep 04 2023 13:25:02 GMT+0100 (British Summer Time),7.22.15,Mon Sep 04 2023 13:25:02 GMT+0100 (British Summer Time),22
+react-router-dom,commonjs,6.16.0,Wed Sep 13 2023 17:37:49 GMT+0100 (British Summer Time),5.3.4,Sun Oct 02 2022 18:10:36 GMT+0100 (British Summer Time),359
+xmldoc,commonjs,1.3.0,Mon Mar 20 2023 21:39:03 GMT+0000 (Greenwich Mean Time),1.3.0,Mon Mar 20 2023 21:39:03 GMT+0000 (Greenwich Mean Time),190
+@babel/plugin-transform-runtime,commonjs,8.0.0-alpha.2,Mon Sep 04 2023 13:25:20 GMT+0100 (British Summer Time),7.22.15,Mon Sep 04 2023 13:25:20 GMT+0100 (British Summer Time),22
+react-router,commonjs,6.16.0,Wed Sep 13 2023 17:37:49 GMT+0100 (British Summer Time),5.3.4,Sun Oct 02 2022 18:10:33 GMT+0100 (British Summer Time),359
+moment,commonjs,2.29.4,Sun Aug 27 2023 12:38:44 GMT+0100 (British Summer Time),2.29.4,Wed Jul 06 2022 17:01:32 GMT+0100 (British Summer Time),447
+react-helmet,commonjs,6.1.0,Sun Sep 25 2022 23:10:24 GMT+0100 (British Summer Time),6.1.0,Mon Jun 08 2020 05:59:07 GMT+0100 (British Summer Time),1205
+@babel/preset-env,commonjs,8.0.0-alpha.2,Sat Sep 16 2023 17:28:44 GMT+0100 (British Summer Time),7.22.15,Mon Sep 04 2023 13:25:28 GMT+0100 (British Summer Time),22
+uuid,commonjs,9.0.1,Tue Sep 12 2023 09:56:35 GMT+0100 (British Summer Time),8.3.2,Tue Dec 08 2020 20:38:36 GMT+0000 (Greenwich Mean Time),1022
+winston,commonjs,3.10.0,Sun Aug 27 2023 12:38:45 GMT+0100 (British Summer Time),Unkonwn,Invalid Date,NaN
```

Comment:
Should we try to code against Invalid dates & NaNs? Or perhaps try to figure out what these values are so we can cater for them?
```suggestion
winston,commonjs,3.10.0,Sun Aug 27 2023 12:38:45 GMT+0100 (British Summer Time),Unkonwn,Invalid Date,NaN
```

Reply:
It was because those dependencies have patch files and so their version number was within a different kind of string. I have added some code that is ale to get the Winston and ramada version numbers, and if in the future there is one like this with a new string structure, it prints out the string so we can look at it to adapt.